user-documentation/workshops/recipe-authoring.md

```diff
@@ -484,39 +415,60 @@ Let's look at an existing imperative recipe in the starter project, and see how
+* Install and configure the Moderne IntelliJ IDEA plugin
+* Perform a type-aware search across all of your local repositories
+* Create a simple search recipe
+* Debug your recipe, and use the `TreeVisitingPrinter` to see what the LST looks like
 
-When creating new recipes, you may find it desirable to examine multiple source files, potentially of different types, to make key decisions in your visitor. For example, you may want to look for a particular condition to be present in a Maven POM file and, if that condition is met, alter an application property in a YAML file. This is where [scanning recipes](https://docs.openrewrite.org/authoring-recipes/writing-recipes-over-multiple-source-file-types) come in.
+#### Steps
 
-The rewrite-recipe starter contains an example in the form of [src/main/java/com/yourorg/AppendToReleaseNotes.java](https://github.com/moderneinc/rewrite-recipe-starter/blob/main/src/main/java/com/yourorg/AppendToReleaseNotes.java) that you might want to explore.
+1. If you haven't already installed the Moderne IntelliJ IDEA plugin, follow along with our [Moderne plugin installation guide](/user-documentation/moderne-ide-integration/how-to-guides/moderne-plugin-install.md)
+   * If you have many repositories checked out locally and want to search across those, please add the root directory as a `Multi-repo`.
+   * If you don't have many repositories checked out locally or would prefer to see what it looks like to add a Moderne organization, please select one of the Moderne organizations (such as `Default` or `Netflix`) in the `Multi-repos` section. Note that if you select `Default`, this is the same `Default` group you used earlier in this workshop.
+2. Open up any Java class in IntelliJ and look for an API that you're interested in searching for (e.g., `System.out.println(..)` or `ListUtils.map(..)`). Then, follow the instructions in our [multi-repository code search doc](/user-documentation/moderne-ide-integration/how-to-guides/code-search.md) to look for that API across all of the repositories you added to the Moderne plugin.
+3. Next, let's create a simple search recipe that finds that API you searched for. Right-click on the API again and select `Refactor > Create OpenRewrite Recipe...`. Then select that you want to create a `Visitor Style` recipe.
+   * For more details on creating recipes with the Moderne plugin, check out our [how to create recipes guide](../moderne-ide-integration/how-to-guides/creating-recipes.md).
+4. You should now have a scratch file that contains a simple recipe. Copy it over to the `rewrite-recipe-starter` repository you were using earlier and add it to the `com.yourorg` package.
+   * [Here's an example of what this might look like for finding System.out.println](https://gist.github.com/mike-solomon/3b49a5d19c8824776bcc4ee871b87cdd)
+5. Re-build the `rewrite-recipe-starter` project and make sure to publish it to your Maven local directory so the CLI can find this new recipe.
```

Comment:
This should not be necessary for the `Set Active Recipe` flow right?
```suggestion
```

Reply:
True - I think I used the recipe name instead of the active recipe flag as I should've been when testing. I'll renumber this and get that updated.